bug: fix mirros pull failed in debian10_1.0.3

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the problem

```
Need to get 4307 kB of archives.
After this operation, 10.1 MB of additional disk space will be used.
Get:1 http://mirrors.ustc.edu.cn/debian buster/main amd64 sensible-utils all 0.0.12 [15.8 kB]
Get:2 http://mirrors.ustc.edu.cn/debian buster/main amd64 ucf all 3.0038+nmu1 [69.0 kB]
Get:3 http://mirrors.ustc.edu.cn/debian buster/main amd64 bc amd64 1.07.1-2+b1 [109 kB]
Get:4 http://mirrors.ustc.edu.cn/debian buster/main amd64 libcurl4 amd64 7.64.0-4+deb10u2 [332 kB]
Get:5 http://mirrors.ustc.edu.cn/debian buster/main amd64 curl amd64 7.64.0-4+deb10u2 [265 kB]
Get:6 http://mirrors.ustc.edu.cn/debian buster/main amd64 fonts-dejavu-core all 2.37-1 [1068 kB]
Get:7 http://mirrors.ustc.edu.cn/debian buster/main amd64 fontconfig-config all 2.13.1-2 [280 kB]
Get:8 http://mirrors.ustc.edu.cn/debian buster/main amd64 lcov all 1.13-4 [128 kB]
Get:9 http://mirrors.ustc.edu.cn/debian buster/main amd64 libpng16-16 amd64 1.6.36-6 [292 kB]
Ign:10 http://mirrors.ustc.edu.cn/debian buster/main amd64 libfreetype6 amd64 2.9.1-3+deb10u2
Get:11 http://mirrors.ustc.edu.cn/debian buster/main amd64 libfontconfig1 amd64 2.13.1-2 [346 kB]
Get:12 http://mirrors.ustc.edu.cn/debian buster/main amd64 libjpeg62-turbo amd64 1:1.5.2-2+deb10u1 [133 kB]
Get:13 http://mirrors.ustc.edu.cn/debian buster/main amd64 libjbig0 amd64 2.1-3.1+b2 [31.0 kB]
Get:14 http://mirrors.ustc.edu.cn/debian buster/main amd64 libwebp6 amd64 0.6.1-2+deb10u1 [261 kB]
Err:15 http://mirrors.ustc.edu.cn/debian-security buster/updates/main amd64 libtiff5 amd64 4.1.0+git191117-2~deb10u4
  404  Not Found [IP: 202.141.1[60](https://github.com/stoneatom/stonedb/actions/runs/4034659868/jobs/6936309975#step:4:61).110 80]
Get:16 http://mirrors.ustc.edu.cn/debian buster/main amd64 libxpm4 amd64 1:3.5.12-1 [49.1 kB]
Get:17 http://mirrors.ustc.edu.cn/debian buster/main amd64 libgd3 amd64 2.2.5-5.2 [136 kB]
Get:18 http://mirrors.ustc.edu.cn/debian buster/main amd64 libgd-perl amd64 2.71-2 [140 kB]
Get:10 http://mirrors.ustc.edu.cn/debian buster/main amd[64](https://github.com/stoneatom/stonedb/actions/runs/4034659868/jobs/6936309975#step:4:65) libfreetype6 amd64 2.9.1-3+deb10u2 [380 kB]
E: Failed to fetch http://mirrors.ustc.edu.cn/debian-security/pool/updates/main/t/tiff/libtiff5_4.1.0+git191117-2~deb10u4_amd64.deb  404  Not Found [IP: 202.141.160.110 80]
Fetched 4036 kB in 18s (223 kB/s)
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
```

### Expected behavior

It should be run successed.

### How To Reproduce

_No response_

### Environment

github
os: debian10_1.0.3

### Are you interested in submitting a PR to solve the problem?

- [X] Yes, I will!

--- a/.github/workflows/pull_requests.yml
+++ b/.github/workflows/pull_requests.yml
@@ -87,7 +87,7 @@ jobs:
         run: |
           case ${{ matrix.os }} in
             debian10_1.0.3)
-              apt-get install curl lcov bc -y
+              apt-get update && apt-get install curl lcov bc -y
               ;;
             centos7_1.0.3)
               yum install curl lcov.noarch bc.x86_64 -y

